frontend/public/components/_storage-class-form.scss

```
@@ -1,3 +1,4 @@
 .create-storage-class-form__checkbox {
   width: inherit;
+  margin-bottom: 0px !important
```

why do you want to make it a forceful override by marking it important?
```suggestion
  margin-bottom: 0px !important;
```

---

ack